src/components/table/ExportXlsx.jsx

```diff
@@ -24,6 +25,20 @@ const ExportXlsx = (props) => {
     exportXlsxModalRef.current.hide();
   };
 
+  if (!useSelector((store) => store.user.paid_subscription_permission)) {
```

Comment:
hooks should be declared at top of component

Reply:
done